request-example.go

```diff
@@ -26,6 +27,7 @@ func InMemHandler() Handlers {
 
 // Example Handlers
 func (fs *root) Fileread(r *Request) (io.ReaderAt, error) {
+	_ = r.WithContext(r.Context())
```

Comment:
I‘m not sure I see the purpose of all of these `_ = r.WithContext(r.Context())` calls. Were they there just for debugging?

Reply:
Issue #280 was totally missed in testing because the Context was never initialized in any test. So I added these to trigger that issue and I decided to leave it in to be sure I wouldn't re-introduce the deadlock by mistake. I tried a couple times to come up with a single/standalone test and I didn't like any of them. So I just kept these as being better than nothing.

Earlier in my career I suffered from being a bit to much of a perfectionist and would over-engineer and take forever to get things out. Now as long as I think it isn't terrible I'll go ahead and get it released knowing it can always be reworked.

Maybe I should add a comment to make sure the intention is clear?

Reply:
I'm just thinking... `_ = r.WithContext(r.Context()) // initialize context for testing` for now.

Reply:
Slight tweak to before... `_ = r.WithContext(r.Context()) // initialize context for deadlock testing` makes it clearer.